pixeltable/share/publish.py

```diff
@@ -10,22 +9,22 @@
 import pixeltable as pxt
 from pixeltable import exceptions as excs
 from pixeltable.env import Env
+from pixeltable.metadata.schema import FullTableMd
 from pixeltable.utils import sha256sum
 
 from .packager import TablePackager
 
 # These URLs are abstracted out for now, but will be replaced with actual (hard-coded) URLs once the
 # pixeltable.com URLs are available.
-_PUBLISH_URL = os.environ.get('PIXELTABLE_PUBLISH_URL')
-_FINALIZE_URL = os.environ.get('PIXELTABLE_FINALIZE_URL')
+
+PIXELTABLE_API_URL = 'https://internal-api.pixeltable.com'
 
 
 def publish_snapshot(dest_tbl_uri: str, src_tbl: pxt.Table) -> str:
-    packager = TablePackager(src_tbl, additional_md={'table_uri': dest_tbl_uri})
+    packager = TablePackager(src_tbl, additional_md={'table_uri': dest_tbl_uri, 'operation_type': 'publish_snapshot'})
```

Comment:
I don't think `operation_type` belongs inside the additional_md struct? Metadata represents state, while operation_type is an action